src/Components/Modal/WinModal.js

```diff
@@ -17,25 +17,39 @@ class WinModal extends Component {
     }
 
     render() {
-        /*var i
-        let players = []
-        for (i = 0; i < this.state.board["num_players"]; i++) {
-            players.push(this.state.board["player_ids"][i])
-        }
 
-        let testing = ""
+        var i
```

Comment:
more descriptive variable names will make the code easier to read. please re-name the variable.

Reply:
or add a comment to describe the variable's purpose